tests/integration/defs/examples/serve/test_serve_negative.py

```
@@ -0,0 +1,605 @@
+    print_info("Testing health check during error conditions...")
+
+    async def send_invalid_requests():
+        """Background task sending invalid requests"""
+        async_client = server.get_async_client()
+        for _ in range(50):
+            try:
+                await async_client.chat.completions.create(
+                    model=model_name,
+                    messages=[],  # Invalid
+                    max_tokens=5)
+            except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)